packages/eslint-plugin/src/configs/base.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "plugins": ["@typescript-eslint"],
+  "rules": {
+    "camelcase": "off",
```

Comment:
these shouldn't be in the base config - they should only be disabled if the corresponding rule is enabled.

I would suggest (for now) just hard code a `Set` into the generation script which automates adding it to the all / recommended configs

Reply:
I moved all rules back, as a Set, like you proposed. Additional I had a look at eslints all.config and saw, they prevent deprecated rules from being applied to their configs. I added this, too, with the ability to explicitly add deprecated rules. Thanks for your great review 😄